src/main/java/org/myrobotlab/service/DiscordBot.java

```diff
@@ -153,21 +163,29 @@ public void setToken(String token) {
     this.token = token;
   }
 
-  public static void main(String[] args) throws Exception {
-    // Brief example of starting a programab chatbot and connecting it to
-    // discord
+  public void runMrT() throws LoginException, IOException {
```

Comment:
this should not be on the DiscordBot.

Reply:
i see what you are referring to.. in the sendUtterance method.. that default logic should be removed and an exception should be thrown if the channel isn't known.

Reply:
that's increasing scope - i lifted what was previously done in sendUtterance

Reply:
I didn't put runMrT in it..  although i agree ... i might remove it